assets/js/tracking.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Function for registering tracking to a link. To add tracking to a link you
+ * need to have an "id" attritube on the <a> element and the "track-click" attribute.
+ *
+ *   - ex. <a id="get-started-no-yaml" class="btn" href="/docs/get-started" track-click>GET STARTED</a>
+ */
+$(document).ready(function() {
+
+    // Check if the analytics object and track function are available. If they are
+    // not we do not even want to attempt to track anything.
+    if (window && window.analytics && typeof window.analytics.track === "function") {
```

Comment:
Where does the `analytics` object come from? Segment? I wonder how this interacts with GDPR and what not. HubSpot pops up a cookies content form, is there a way we can read that setting?

It might be worth just filing a new issue in the `pulumi/home` repo to track following up on how the "cookie content" issue is managed, and how we can be consistent in honoring that option on our web properties.

Reply:
Created an issue here: https://github.com/pulumi/home/issues/608

Assigned it to both of us for tracking. Do we have a lawyer we can reach out to about this?